Classes/Systems/ShortcutHandler.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ShortcutHandler.swift
+//  Freetime
+//
+//  Created by Viktor Gardart on 2017-10-08.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+struct ShortcutHandler {
+
+    private enum Items: String {
+        case search
+        case switchAccount
+    }
+
+    static func configure(application: UIApplication, sessionManager: GithubSessionManager) {
+        application.shortcutItems = generateItems(sessionManager: sessionManager)
+    }
+
+    static func handle(shortcutItem item: UIApplicationShortcutItem,
+                       sessionManager: GithubSessionManager,
+                       navigationManager: RootNavigationManager) -> Bool {
+        let itemType = Items(rawValue: item.type)
+        switch itemType {
+        case .some(.search):
+            navigationManager.selectViewController(atIndex: 1)
```

Comment:
Maybe change this later when we support urlschemes, `githawk://search` or something.

Reply:
Maybe add a constant referencing search is at index 1 atm?